src/util/loanUtils.js

```diff
@@ -75,6 +75,22 @@ export function buildPriceArray(amountLeft, minAmount) {
 	return priceArray;
 }
 
+export function build5DollarsPriceArray(amountLeft) {
```

Comment:
This function should get a few unit tests. It is an easily testable method, and this experiment has a lot of app implications, so it would be good to get some stability in before release.